package.json

```diff
@@ -4,6 +4,15 @@
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
+  "exports": {
+    ".": "./lib/index.js",
```

Comment:
is it worth being a little forward thinking here and using the 

```
{ "default": "./lib/index.js" } 
```
configuration, which might allow for separately supporting an esm and cjs build at some point?

Reply:
sure I can change this